style/infra_template.rubocop.yml

```diff
@@ -0,0 +1,87 @@
+## Infrastructure .rubocop.yml template
+
+require:
+  - rubocop-rails
+  - rubocop-rspec
+
+inherit_from: .rubocop_todo.yml
+
+AllCops:
+  TargetRubyVersion: 3.1
+  SuggestExtensions: false
+  DisplayCopNames: true
+  Include:
+    - './Rakefile'
+    - './config.ru'
+    - '**/*.rb'
+    - '**/*.rake'
+  Exclude:
+    # autogenerated files (that we don't change) should go here
+    - 'bin/**/*'
+    - 'config/environments/*.rb'
+    - 'config/routes.rb'
+    - 'db/**/*'
+    - 'spec/spec_helper.rb'
+    - 'spec/rails_helper.rb'
+    - 'vendor/**/*'
+
+Bundler/GemComment:
+  OnlyFor:
+    - restrictive_version_specifiers
+    - branch
+
+# Configuration parameters: AllowURI, URISchemes.
+Layout/LineLength:
+  Max: 120
+
+Metrics/BlockLength:
+  IgnoredMethods:
+    - SimpleForm.setup
+    - FactoryBot.define
+    - factory
+    - describe
+    - context
+
+# consider if refactoring will improve adherence to SOLID and/or improve readability
+#  Sometimes splitting the class up won't improve readablity, it just makes reader jump around in the code.
+Metrics/ClassLength:
+  Max: 150 # default 100
+
+# consider if refactoring will improve adherence to SOLID and/or improve readability
+#  Sometimes splitting the method up won't improve readablity, it just makes reader jump around in the code.
+#  Good method names help.
+Metrics/MethodLength:
+  Max: 15 # default 10
+
+RSpec/DescribeClass:
+  Exclude:
+    - 'spec/views/**/*_spec.rb' # In view tests, the view is named, not the class
+
+# do try to keep examples short (can you split out the setup from the test?)
+RSpec/ExampleLength:
+  Enabled: false
+
+# most tests should test a single thing
+RSpec/MultipleExpectations:
+  Max: 5 # default 1
+  Exclude:
+    - 'spec/features/**/*_spec.rb'
+
+RSpec/MultipleMemoizedHelpers:
+  Enabled: false
+
+RSpec/NestedGroups:
+  Max: 4 # default: 3
+
+# "> 0" is not less readable than ".positive?", "== 0" than ".zero?", etc.
+Style/NumericPredicate:
+  Enabled: false
```

Comment:
I don't want to die on this hill... but I like numeric predicates, myself.

Reply:
can this become a discussion item in dev planning?  I'm willing to bend, but let's poll for the will of the people.

Reply:
Fine by me. :)

Reply:
Happy for this to be a discussion. My 2 cents, I much prefer `.positive?` and `.zero?` predicates and disagree with the notion that they are not more readable than `> 0`... My preference is to enforce "idiomatic ruby" as much as possible (for whatever measure that is), but like @mjgiarlo I won't die on this hill.

Reply:
@sul-dlss/infrastructure-team If I were pressed to try to convince someone to lean in the direction of `#positive?` / `#zero?`, I might argue the following:

1. I love OO in Ruby, and if I have to choose between calling a method and using built-in keywords and operators, I default to the former. (See also: using `Enumerable#each` over `for` or `while` loops.)
2. There's less chance of an assignment / equality goof when using `number.zero?` vs. `number == 0`.

But I don't see either as a slam dunk, really. 🤷🏻‍♂️

Reply:
No major preference, but the second point lands with me, given that I just fixed a bug a couple weeks ago that was an assignment/equality problem.

Reply:
We decided as a team to use the rubocop default here.